<launch>

  <node name="serial_node" pkg="kobuki_ui" type="serial_node.py" output="screen">
  </node>

  <node name="station_server" pkg="kobuki_ui" type="station_server.py" output="screen">
  </node>


  <node name="queue_move" pkg="kobuki_ui" type="queue_move.py" output="screen">
  </node>
</launch>
